src/core/components/Paginate/Paginate.scss

```diff
@@ -11,16 +11,21 @@
   justify-content: space-between;
 }
 
+.Paginate {
+  padding: 0 10%;
+  margin-bottom: 5%;
+}
+
 .Paginate .Paginate-item {
   @include button($background: #fff, $color: #000);
```

Comment:
I know this isn't part of your patch, but could you change this line to use colour variables too?

Actually this should be using your colour that you used below. It should be: `button($background: $base-color, $color: $link-color);`.